lib/presentation/bloc/now_playing_movies_bloc.dart

```diff
@@ -0,0 +1,30 @@
+import 'dart:async';
+
+import '../../core/bloc/bloc.dart';
+
+import '../../data/model/data_state.dart';
+
+import '../../domain/usecase/implementation/get_now_playing_movies_usecase.dart';
+import '../../domain/usecase/usecase_interface.dart';
+
+class NowPlayingMoviesBloc implements Bloc {
+  UseCase useCase = GetNowPlayingMoviesUseCase();
```

Comment:
Same as above

Reply:
All the blocs has almost the same functionality, you can create only one, an use add all the UseCases like this
```
MoviesBloc({
    required this.nowPlayingUseCase,
    required this.popularUseCase,
    ....
});

final GetNowPlayingUseCase nowPlayingUseCase;
final GetPopularUseCase popularUseCase;
```

Reply:
done...DRY! again!